src/DataHandling/XMLExtractor.php

```diff
@@ -0,0 +1,148 @@
+<?php
+namespace CubeTools\CubeCommonBundle\DataHandling;
+
+use Symfony\Component\DomCrawler\Crawler;
+
+class XMLExtractor
+{
+    /**
+     * @var \Symfony\Component\DomCrawler\Crawler
+     */
+    protected $crawler;
+    
```

Comment:
many trailing whitespaces (Lines 12, 17, 22, ...).
please run `composer install` (or update) to install the pre-commit-hook

The hook will show many more style failures, I do not list all here.

You can also run it manually with
```
./vendor/cubetools/cube-common-develop/src/CodeStyle/check-commit-cube.sh --changed
```
or
```
./vendor/cubetools/cube-common-develop/src/CodeStyle/check-commit-cube.sh HEAD~2
```